provisioning/roles/php/tasks/main.yml

```diff
@@ -50,7 +50,7 @@
   when: ansible_os_family == "Debian"
 
 - name: Customized memory limits
-  copy: dest={{ php_mods_available }}/memory_limits.ini content="; configuration for memory limits\nmemory_limit=512M\nupload_max_filesize=1024M\npost_max_size=1024M\n"
+  copy: dest={{ php_mods_available }}/memory_limits.ini content="; configuration for memory limits\nmemory_limit = 160M\nupload_max_filesize = 1024M\npost_max_size = 1024M\n"
```

Comment:
Definitely not recommended in any kind of real-life production with preforked PHP. Experience shows that 128M is okay, even 160M is rarely needed. 128 conns x 256M is really quite a lot even on mid-range servers, let alone 512M. Avoid doing that for non-CLI PHP, and even for CLI I haven't had a case where it's absolutely required.